app/models/remote_execution_feature.rb

```diff
@@ -24,8 +24,10 @@ def self.feature(label)
   end
 
   def self.register(label, name, options = {})
+    pending_migrations = ::Foreman::Plugin.registered_plugins[:foreman_remote_execution]&.pending_migrations
```

Comment:
Just a nitpick, it seems better to use `find` as it feels like a real public interface.
```suggestion
    pending_migrations = ::Foreman::Plugin.find(:foreman_remote_execution)&.pending_migrations
```

Reply:
You didn't give me time to actually apply this

Reply:
I didn't think it's important to change now, I don't see the `registred_plugins` going away anytime soon :)
but sorry, will wait for a bit next time ;)